plugins/bulk-import/src/components/BulkImportIcon.tsx

```diff
@@ -15,8 +15,6 @@ export const icon = () => {
 };
 
 export const BulkImportIcon = () => {
-  // permission logic
-
   return (
     <SidebarItem text="Bulk import" to="bulk-import/repositories" icon={icon} />
   );
```

Comment:
As mentioned in another comment. BulkImportIcon should return an Icon or should have another name.

The name confused me. If you fix this, please also update the README. :)

I'm fine if you want to open another PR for that.